src/pages/portfolio-page/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Container, Box, makeStyles, Card, useTheme, useMediaQuery } from '@material-ui/core';
+
+import { DetailsCard } from '../../components/DetailsCard';
+import { SectionTitle } from '../../components/SectionTitle';
+import { PortfolioCard } from '../../components/PortfolioCard';
+
+import { projectData, userData } from './data';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    padding: 0,
+    [theme.breakpoints.up('lg')]: {
+      display: 'flex',
+      minHeight: '100vh',
```

Comment:
Why you need this `minHeight`?

Reply:
It was for development purposes and I forgot to remove, thanks for catching